expr/verdict.go

```diff
@@ -126,3 +135,37 @@ func (e *Verdict) unmarshal(data []byte) error {
 	}
 	return ad.Err()
 }
+
+func (e *Verdict) String() string {
+	var v string
+	switch e.Kind {
+	case unix.NFT_RETURN:
+		v = "return" // -0x5
```

Comment:
What did you want to express with these comments?

Reply:
> What did you want to express with these comments?

@stapelberg , just a note as to what the id is for the return.

Reply:
If you follow my other suggestion above regarding using the `Verdict*` names, I think it’d be cleaner to not duplicate these values (we only want a single place to update, not multiple). If people are inclined to find out the value, they should jump to the definition using their editor.